db/1.10/modules/ROOT/pages/database-documentation.adoc

```diff
@@ -676,9 +676,12 @@ This operation selects data from a database. Streaming is automatically applied
 === Working with Pooling Profiles
 When working with pooling profiles and the *Select* operation, the connection remains open until one of the following occurs:
 
-* The flow execution ends 
+* The flow execution ends
 * The content of the streams are consumed completely
-* The connection is the transaction key. 
+* The connection is the transaction key.
+
+[NOTE]
+Because LOBs are treated as streams, the connection remains open until the flow execution ends, or until the content is consumed before the flow completes, in which case the best approach is taken to close the related connection.
```

Comment:
I would change it to: "Because LOBs are treated as streams, the connection remains open until the flow execution ends or until the content is completed consumed before the flow finishes. A best-effort approach is taken to handle the connections."